packages/astro/package.json

```diff
@@ -78,7 +78,7 @@
       "default": "./dist/core/middleware/namespace.js"
     },
     "./transitions": "./dist/transitions/index.js",
-    "./transitions": "./dist/transitions/index.js",
+    "./transitions/router": "./dist/transitions/router.js",
```

Comment:
This was a an error of mine when I rebased the `i18n-routing` branch with `main`